alvr/client_openxr/src/lib.rs

```diff
@@ -282,9 +284,20 @@ pub fn entry_point() {
             default_view_resolution,
             &last_lobby_message,
         );
+        let lobby_body_config = BodyTrackingSourcesConfig {
+            body_tracking_fb: Switch::Disabled,
+            body_tracking_bd: if platform.is_pico() {
+                Switch::Enabled(BodyTrackingBDConfig {
+                    high_accuracy: true,
+                    prompt_calibration_on_start: false,
+                })
+            } else {
+                Switch::Disabled
+            },
```

Comment:
I don't think we need to filter by pico because the feature is already pico specific